data/json/overmap/overmap_special/specials.json

```diff
@@ -8023,5 +8023,168 @@
+      { "point": [ 2, 3, -1 ], "overmap": "retirement_community_park_b_4_north" },
+      { "point": [ 3, 3, 0 ], "overmap": "retirement_community_park_5_north" },
+      { "point": [ 3, 3, -1 ], "overmap": "retirement_community_park_b_5_north" },
+      { "point": [ 3, 3, 1 ], "overmap": "retirement_community_park_2f_5_north" },
+      { "point": [ 4, 3, 0 ], "overmap": "retirement_community_park_6_north" },
+      { "point": [ 4, 3, -1 ], "overmap": "retirement_community_park_b_6_north" },
+      { "point": [ 2, 4, 0 ], "overmap": "retirement_community_park_7_north" },
+      { "point": [ 2, 4, -1 ], "overmap": "retirement_community_park_b_7_north" },
+      { "point": [ 3, 4, 0 ], "overmap": "retirement_community_park_8_north" },
+      { "point": [ 3, 4, -1 ], "overmap": "retirement_community_park_b_8_north" },
+      { "point": [ 4, 4, 0 ], "overmap": "retirement_community_park_9_north" },
+      { "point": [ 4, 4, -1 ], "overmap": "retirement_community_park_b_9_north" },
+      { "point": [ 3, 7, 0 ], "overmap": "retirement_community_road_north" },
+      { "point": [ 4, 7, 0 ], "overmap": "retirement_community_sub_station_west" },
+      { "point": [ 4, 7, 1 ], "overmap": "sub_station_roof_west" },
+      { "point": [ 4, 7, -1 ], "overmap": "retirement_community_sewer_sub_station_west" },
+      { "point": [ 4, 7, -2 ], "overmap": "underground_sub_station" },
+      { "point": [ 2, 7, 0 ], "overmap": "s_lot_east" },
+      { "point": [ 1, 1, -1 ], "overmap": "sewer_es" },
+      { "point": [ 2, 1, -1 ], "overmap": "sewer_ew" },
+      { "point": [ 3, 1, -1 ], "overmap": "sewer_esw" },
+      { "point": [ 4, 1, -1 ], "overmap": "sewer_ew" },
+      { "point": [ 5, 1, -1 ], "overmap": "sewer_sw" },
+      { "point": [ 1, 2, -1 ], "overmap": "sewer_ns" },
+      { "point": [ 5, 2, -1 ], "overmap": "sewer_ns" },
+      { "point": [ 1, 3, -1 ], "overmap": "sewer_nes" },
+      { "point": [ 5, 3, -1 ], "overmap": "sewer_nsw" },
+      { "point": [ 1, 4, -1 ], "overmap": "sewer_ns" },
+      { "point": [ 5, 4, -1 ], "overmap": "sewer_ns" },
+      { "point": [ 1, 5, -1 ], "overmap": "sewer_ne" },
+      { "point": [ 2, 5, -1 ], "overmap": "sewer_esw" },
+      { "point": [ 3, 5, -1 ], "overmap": "sewer_new" },
+      { "point": [ 4, 5, -1 ], "overmap": "sewer_esw" },
+      { "point": [ 5, 5, -1 ], "overmap": "sewer_wn" },
+      { "point": [ 2, 6, -1 ], "overmap": "sewer_ns" },
+      { "point": [ 2, 7, -1 ], "overmap": "sewer_ne" },
+      { "point": [ 3, 7, -1 ], "overmap": "sewer_esw" }
+    ],
+    "connections": [
+      { "point": [ 3, 8, 0 ], "terrain": "road", "connection": "local_road", "from": [ 3, 7, 0 ] },
+      { "point": [ 3, 7, -1 ], "terrain": "sewer", "connection": "sewer_tunnel", "from": [ 3, 8, -1 ] },
+      { "point": [ 4, 8, -2 ], "terrain": "subway", "connection": "subway_tunnel", "from": [ 4, 9, -2 ] }
+    ],
+    "city_distance": [ -1, 1 ],
```

Comment:
I'd bump up the max city distance to 5 at least - it's a 7x7 special, and if any of the points overlaps a city tile it will bail out of placement.

Reply:
Fair point, I will make that change.